README.md

```diff
@@ -61,3 +69,19 @@ test('Automated accessibility testing', async t => {
     await t.expect(violations.length === 0).ok(createReport(violations));
 });
 ```
+
+## Using full axe result object and axe.configure
+
+If you prefer to use a custom reporter for axe results you can get result object using runAxe function:
+
+```js
+import { runAxe } from '@testcafe-community/axe';
+
+fixture `TestCafe tests with Axe`
+    .page `http://example.com`;
+
+test('Automated accessibility testing', async t => {
+    const { error, results } = await axeCheck(t);

```

Comment:
In the import you import runAxe but here you're using checkAxe... Is that right?

Reply:
Great catch! I actually intended to use runAxe in the example. Fixing.